pkg/services/dashboards/database/database_folder_test.go

```diff
@@ -230,197 +214,6 @@ func TestIntegrationDashboardFolderDataAccess(t *testing.T) {
 	})
 }
 
-func TestIntegrationDashboardInheritedFolderRBAC(t *testing.T) {
```

Comment:
moved to TestIntegrationDashboardServicePermissions (in pkg/tests/api/dashboards) - also needed grafana running now